x/bridge/keeper/assets_locked_test.go

```diff
@@ -205,6 +205,102 @@ func TestAcceptAssetsLocked(t *testing.T) {
 			},
 			errContains: "failed to send coins",
 		},
+		{
+			name: "BTC mint to blocked address as recipient",
+			bankKeeperFn: func(ctx sdk.Context) *mockBankKeeper {
+				bankKeeper := newMockBankKeeper()
+
+				bankKeeper.On(
+					"MintCoins",
+					ctx,
+					types.ModuleName,
+					sdk.NewCoins(
+						sdk.NewCoin(evmtypes.DefaultEVMDenom, math.NewInt(1)),
+					),
+				).Return(nil)
+
+				bankKeeper.On(
+					"SendCoinsFromModuleToAccount",
+					ctx,
+					types.ModuleName,
+					mock.Anything,
+					mock.Anything,
+				).Return(nil)
```

Comment:
Let's return an error from `SendCoinsFromModuleToAccount` that will emphasize what happens in reality.

Reply:
Not sure what you mean exactly seeing that we expect this call to never be called, which assert just after?

Reply:
Right, I somehow missed the assertion.